### DISCUSSION TITLE
Transmission LXC hangs

### DISCUSSION THREAD

**Discussion by thealphonso** [tteck/Proxmox#1719] [Help Needed]
Every couple of days I notice that the Transmission LXC hangs and I cannot connect to it anymore. The memory usage is at maximum (both swap and RAM); also the CPU usage is around 50%.
Allocated resources are CPU: 2, RAM 4GB, Swap: 512MB on an Intel NUC 11 running Proxmox 8.0.4.
Any thoughts on this?

**Comment by nfribeiro** [comment]
I use transmission and haven't had any issue  with it. Did you reinstall it?

**Comment by burgerga** [comment]
Had the same issue, but transmission via tteck uses Debian 12, and is therefore stuck at v3 (see https://github.com/tteck/Proxmox/discussions/1040). 
I installed v4 (I linked the instructions in the discussion linked above), which should have some performance improvements. If you don't hear from me again, consider the hangs solved ;)
